contracts/test/ChainLinkOracle.t.sol

```diff
@@ -2,41 +2,51 @@
 pragma solidity 0.6.11;
 pragma experimental ABIEncoderV2;
 
-import { TestUtil } from "../../../../test/TestUtil.sol";
+import { DSTest } from "../../modules/ds-test/src/test.sol";
+import { SecurityAdmin } from "./accounts/SecurityAdmin.sol";
+import { Commoner } from "./accounts/Commoner.sol";
+import { ChainlinkOracle } from "../ChainlinkOracle.sol";
+
+contract ChainlinkOracleTest is DSTest {
+
+    ChainlinkOracle   orcl;
+    SecurityAdmin   sadmin;
+    Commoner           cam;
```

Comment:
We actually don't need a commoner, we can just instantiate a `fakeFoo` when needed.